src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -324,6 +326,39 @@ bool runtime::intelgpu::IntelGPUBackend::compile(shared_ptr<Function> func)
                                      reversed_axes);
             }
         }
+        else if ("Concat" == op->description())
+        {
+            if (op->get_inputs().empty() || op->get_outputs().empty())
+            {
+                arguments_check(op, 1, 1);
+            }
+            const size_t cldnn_tensor_dims_lim = 4;
+            const size_t ngraph_tensor_dims = get_input_shape(op, 0).size();
+            const shared_ptr<op::Concat> concat_op = static_pointer_cast<op::Concat>(op);
+            const size_t ngraph_concat_axis = concat_op->get_concatenation_axis();
+            vector<string> inputs;
+
+            if (ngraph_tensor_dims > cldnn_tensor_dims_lim)
```

Comment:
You don't need this. Everything checks you need are in `IntelGPULayout::get_cldnn_axis`